bindings/2.2074/GeometryDash.bro

```diff
@@ -6,11 +6,11 @@ class AccountHelpLayer : GJDropDownLayer, GJAccountDelegate, FLAlertLayerProtoco
 
     void doUnlink();
     void exitLayer();
-    void onAccountManagement(cocos2d::CCObject* sender) = m1 0xb7bf0, imac 0xcefa0;
+    void onAccountManagement(cocos2d::CCObject* sender) = m1 0xb7bf0, imac 0xcefa0, win 0x7fa10;
     void onReLogin(cocos2d::CCObject* sender);
-    void onUnlink(cocos2d::CCObject* sender) = m1 0xb7cc4, imac 0xcf070;
-    void updatePage() = m1 0xb7e88, imac 0xcf230;
-    void verifyUnlink() = m1 0xb81d4, imac 0xcf580;
+    void onUnlink(cocos2d::CCObject* sender) = m1 0xb7cc4, imac 0xcf070, win 0x7fb20;
+    void updatePage() = m1 0xb7e88, imac 0xcf230, win 0x7fcd0;
+    void verifyUnlink() = m1 0xb81d4, imac 0xcf580, win 0x7ffa0;
```

Comment:
verifyUnlink() is inlined on Windows.
the provided address is for FLAlert_Clicked